backport/backport.ts

```diff
@@ -251,9 +251,12 @@ const backport = async ({
 	sender,
 }: BackportArgs) => {
 	const payload = context.payload as EventPayloads.WebhookPayloadPullRequest
-	let payloadLabel = typeof payload.label?.name === 'string' ? payload.label.name : ''
-	if (!(labelRegExp.test(payloadLabel) || backportLabels.includes(payloadLabel))) {
-		return
+	console.log('payloadAction: ' + payload.action)
+	if (payload.action !== 'closed') {
+		let payloadLabel = typeof payload.label?.name === 'string' ? payload.label.name : ''
+		if (!(labelRegExp.test(payloadLabel) || backportLabels.includes(payloadLabel))) {
+			return
+		}
```

Comment:
Might be tired but don't you want to match against all labels, e.g. `let labelsString = labels.map(({ name }) => name)` on the PR as below rather than only when state is not closed and checking `payload.label` ? I think `payload.label` only ever populated on the actions `labeled` and `unlabeled`, but might be wrong. 

Some tests would definitely be good to understand all of this code :)

Reply:
Nah this check should only happen on new labels, for ‘labeled’ actions.
Thought about what you say before, but I get some weird behaviour where some checks don’t happen 🤷🏻‍♂️

Reply:
I will try to move this to Go, since we already have an ongoing project about it.